pkg/cassandra/config/config.go

```diff
@@ -29,21 +30,21 @@ import (
 
 // Configuration describes the configuration properties needed to connect to a Cassandra cluster
 type Configuration struct {
-	Servers              []string       `validate:"nonzero" mapstructure:"servers"`
+	Servers              []string       `validate:"nonzero" mapstructure:"servers" valid:"required,url"`
```

Comment:
do we know what would be using the `validate:` tags? Maybe we should convert them to `valid:`?

Reply:
go-playground/validator is using the `validate:` tags. Since we are using asaskevich/govalidator, we should change this to `valid:`.